test/integration/testdata/egress-proxy.yaml.tmpl

```diff
@@ -48,18 +58,12 @@ spec:
               apiVersion: v1
               fieldPath: metadata.namespace
         volumeMounts:
-        - name: config-volume
```

Comment:
Won't this conflict kube-inject initializer and istioctl kube-inject, who still rely on the configMap? @ayj

Reply:
The egress/ingress proxy are exempt from injection (see ` policy.sidecar.istio.io: "policy.sidecar.istio.io/force-off"`)